spot-contracts/contracts/PerpetualTranche.sol

```diff
@@ -790,21 +860,17 @@ contract PerpetualTranche is
         // We compute "price" as the value of a unit token.
         // The perp, tranche tokens and the underlying are denominated as fixed point numbers
         // with the same number of decimals.
+        IERC20Upgradeable underlying_ = _reserveAt(0);
```

Comment:
Need the trailing underscore? I normally associate that with contract vars

Reply:
We have already method called `underlying` which causes a conflict. Prepending `_` usually indicates private variables or methods.